src/Symfony/Bundle/FrameworkBundle/Command/DebugAutowiringCommand.php

```diff
@@ -162,4 +165,33 @@ private function getFileLink(string $class): string
 
         return (string) $this->fileLinkFormatter->format($r->getFileName(), $r->getStartLine());
     }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        $builder = $this->getContainerBuilder($this->getApplication()->getKernel());
+        $serviceIds = $this->getServices($builder);
+
+        $searchNormalized = $this->getSearchNormalized($input->getArgument('search'));
+
+        $serviceIds = array_filter($serviceIds, function ($serviceId) use ($searchNormalized) {
```

Comment:
I think the class names (service IDs) don't need to be normalized for completion. Suggestions would be identical to defined service IDs.

Reply:
Thank you @GromNaN for the review.

I removed the completion normalization for class names.
I also added the class alias into the completion since the search argument can also take an alias in value.